lib/grell/crawler.rb

```diff
@@ -56,13 +57,15 @@ def start_crawling(url, options = {}, &block)
     def crawl(site, block)
       Grell.logger.info "Visiting #{site.url}, visited_links: #{@collection.visited_pages.size}, discovered #{@collection.discovered_pages.size}"
       site.navigate
+      site.rawpage.page.evaluate_script(@run_before) if @run_before
```

Comment:
I'm scared we will create a lot of javascript errors in the apps if they fail to evaluate the script.